src/JetReconstruction.jl

```diff
@@ -89,4 +89,8 @@ export jetsplot, animatereco
 include("JSONresults.jl")
 export FinalJet, FinalJets
 
+# Substructure modules
+include("Substructure.jl")
+export MassDropTagger, SoftDropTagger, Filter, Trim, massDrop, softDrop, jetFilter, jetTrim
```

Comment:
`Filter` and `Trim` are way too generic names to export into the global namespace. They should be called `JetTrim` and `JetFilter` IMO.

We should stick to the standard Julia naming schemes for methods, lower case and with `_` if needed. So `massDrop` -> `massdrop`, etc.